scripts/lib/CIME/XML/env_batch.py

```diff
@@ -124,7 +124,7 @@ def create_job_groups(self, batch_jobs):
                 self.make_child("type", root=node, text="char")
 
             for child in childnodes:
-                self.add_child(child, root=new_job_group)
+                self.add_child(self.copy(child), root=new_job_group)
```

Comment:
We need separate copies for each job, otherwise we have to deal with aliasing.

Reply:
didn't you just move this from line 115?

Reply:
Yes